vowpalwabbit/reductions/cb/cb_adf.h

```diff
@@ -3,30 +3,22 @@
 // license as described in the file LICENSE.
 
 #pragma once
-#include "io/logger.h"
+#include "cb.h"
 #include "vw_fwd.h"
 
 #include <vector>
 
-namespace CB
-{
-struct cb_class;
-}
-
 namespace VW
 {
-namespace io
+namespace reductions
 {
-struct writer;
+VW::LEARNER::base_learner* cb_adf_setup(VW::setup_base_i& stack_builder);
 }
 }  // namespace VW
 
-VW::LEARNER::base_learner* cb_adf_setup(VW::setup_base_i& stack_builder);
-
+// TODO: Move these functions into VW lib and not reductions
```

Comment:
is this a follow up todo?

Reply:
Yeah, not sure if its what we want or not but we need to think about where cb related library functionality lives. Under reductions or somewhere more general